harness/determined/common/util.py

```diff
@@ -48,6 +48,14 @@ def get_default_master_address() -> str:
     return os.environ.get("DET_MASTER", os.environ.get("DET_MASTER_ADDR", "localhost:8080"))
 
 
+def get_initial_user() -> Optional[str]:
```

Comment:
can we call this `get_container_user()` or something?  `initial` doesn't really explain that it's taking the username of who launched the container.